internal/logger/logger.go

```diff
@@ -63,3 +70,11 @@ func (l *Logger) VerboseErrf(color Color, s string, args ...interface{}) {
 		l.Errf(color, s, args...)
 	}
 }
+
+func EnvColor(envKey string, defaultColor color.Attribute) color.Attribute {
+	envColor, err := strconv.Atoi(os.Getenv(envKey))
+	if err != nil {
+		return defaultColor
+	}
+	return color.Attribute(envColor)
```

Comment:
I think that switching the control flow would make more sense for me:

```suggestion
	envColor, err := strconv.Atoi(os.Getenv(envKey))
	if err == nil {
		return color.Attribute(envColor)
	}
	return defaultColor
```